gensim/models/base_any2vec.py

```diff
@@ -282,9 +282,11 @@ def save(self, fname_or_handle, **kwargs):
 
 
 class BaseWordEmbeddingsModel(BaseAny2VecModel):
-    """
-    Base class containing common methods for training, using & evaluating word embeddings learning models.
-    For example - `Word2Vec`, `FastText`, etc.
+    """Base class containing common methods for training, using & evaluating word embeddings learning models.
```

Comment:
Don't forget about docstrings for `BaseAny2VecModel`.

Also, don't forget about `_` methods, this is really important for persons, who will add some *2vec implementations in future.